packages/terra-form-input/src/Input.jsx

```diff
@@ -108,6 +114,7 @@ class Input extends React.Component {
     const formInputClassNames = cx([
       'form-input',
       { 'form-error': isInvalid },
+      { 'form-incomplete': (isIncomplete && required) },
```

Comment:
What happens if the component is both invalid and incomplete. Is the background yellow with a red border? The callouts show each state separately, but in code they are not mutually exclusive. Can/should a component be in both states at once?

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2670/commits/418492f37795d6dfbe0171c38fef54c1f5fd7577#diff-b91208387be3780b68c9422e87168c2fR118
Might be swapping to allow both, waiting on confirmations.
Ignore the console lines, forgot they were in there and removed them on the next commit.